src/git-repo.ts

```diff
@@ -18,8 +15,17 @@ export class GitRepo {
     this.git = git
   }
 
-  async isInitialized(): Promise<boolean> {
-    return await this.git.checkIsRepo()
+  isInitialized(): boolean {
+    try {
+      // Make sure the string we will pass to to the shell is an actual dir
+      if (!existsSync(this.dir.getDirPath())) {
+        throw new Error()
+      }
+      execSync(`cd ${this.getDirPath()} && git status`)
```

Comment:
I'm not sure if this works with a relative dir path. I think we should not rely on the current working dir. Maybe we can add one more test with a relative path for the `isInitialized()` method.

Reply:
A relative path is accepted now that I changed the GPG key fixture load, so that is based on the loader script path, instead of the cwd.